pkg/clustertask/clustertask.go

```diff
@@ -117,3 +119,44 @@ func getV1alpha1(c *cli.Clients, clustertaskname string, opts metav1.GetOptions)
 	}
 	return clustertask, nil
 }
+
+func Create(c *cli.Clients, ct *v1beta1.ClusterTask, opts metav1.CreateOptions) (*v1beta1.ClusterTask, error) {
+	gvr, err := actions.GetGroupVersionResource(clustertaskGroupResource, c.Tekton.Discovery())
+	if err != nil {
+		return nil, err
+	}
+
+	if gvr.Version == "v1alpha1" {
+		v1alpha1ClusterTask := convertToV1Alpha1(ct)
+		return createUnstructured(v1alpha1ClusterTask, c, opts)
+	}
+
+	return createUnstructured(ct, c, opts)
+}
+
+func createUnstructured(obj runtime.Object, c *cli.Clients, opts metav1.CreateOptions) (*v1beta1.ClusterTask, error) {
+	object, _ := runtime.DefaultUnstructuredConverter.ToUnstructured(obj)
```

Comment:
Does this return an error as well here, if yes then i think we should handle that